### ISSUE TITLE
32-bit error message is sometimes misleading

### ISSUE THREAD

**Issue by fagerbua** [amilajack/eslint-plugin-flowtype-errors#102]
The changes in #46 makes the plugin fail when Flow doesn't run. That is clearly an improvement. But the error message is not always accurate: it complains about 32 bit even when the cause is something else — for example, the glibc version being too low ([Flow issue #49](https://github.com/flowtype/flow-bin/issues/49)), or the `FLOW_BIN` variable pointing to a binary that doesn't exist.

As a solution I would propose changing the error message to say simply that the plugin was unable to run Flow. If the 32-bit error is very common it might still be mentioned as a possible cause.

If you agree I could try to prepare a PR for this change.

**Comment by amilajack** [comment]
@fagerbua i agree with this. A PR would be welcome. Mentioning likely issues with why flow might not be running would be a nice improvement. Ex:
```bash
Flow could not be run. Common issues include
* running on 32-bit OS
* glibc not found (https://github.com/flowtype/flow-bin/issues/49)
```

**Comment by jdmota** [comment]
Sorry about that. Didn't know about those cases.

For the PR, you just need to change the message here:

https://github.com/amilajack/eslint-plugin-flowtype-errors/blob/2586b6f524f1b36b56887d5fa5c9d43262bad170/src/index.js#L48-L53
